apps/user/src/app/(main)/map/components/NaverMap.tsx

```
@@ -0,0 +1,41 @@
+import { useCallback, useRef, useEffect } from "react";
+import { Coordinates } from "@/types/map";
+import type { NaverMap, NaverMapOptions, NaverLatLng } from "@/types/map";
+
+const mapId = "naver-map";
+
+interface NaverMapProps {
+  loc: Coordinates;
+  zoom?: number;
+}
+
+export default function NaverMap({ loc, zoom = 15 }: NaverMapProps) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'NaverMap'. Consider to delete it or rename it.

'NaverMap' is defined here:


(lint/suspicious/noRedeclare)